api/jest.config.ts

```diff
@@ -1,11 +1,20 @@
+import path from 'node:path';
 import type { Config } from 'jest';
 
+import { config as envConfig } from 'dotenv';
+
+const envPath = path.resolve(__dirname, '../.env');
+envConfig({ path: envPath });
+
 const config: Config = {
   verbose: true,
   testRegex: '\\.test\\.ts$',
   transform: {
     '^.+\\.ts$': 'ts-jest'
-  }
+  },
+  ...(process.env.FREECODECAMP_NODE_ENV == 'development'
+    ? { maxWorkers: 5 }
```

Comment:
```suggestion
    ? { maxWorkers: NODE_MAX_ENV ||  whateverisdefault }
```

Could we do something like this?

Reply:
I decided to just remove this from the PR - I will make another specific for it.